CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## [Unreleased]
+- Changed handling of `NavigationService` with its rerouting, etc. events, now it's not in `MapboxVisionAR` module
```

Comment:
Now it's not in `X` module. That's OK, but where it is now? Should we clarify it in changelog?

Reply:
Changed